libs/remix-simulator/src/methods/misc.ts

```diff
@@ -0,0 +1,62 @@
+const version = require('../../package.json').version
+import web3 from 'web3'
+
+export class Misc {
```

Comment:
For me it doesn't make sense to create a class. Create a class if either : 
- You need a state
- or you need to have multiple instances.

In this case I've got the feeling you don't need it. You're just wrapping methods here : 
```typescript
export function web3ClientVersion(payload, cb) {
  cb(null, 'Remix Simulator/' + version)
}
export function ethProtocolVersion(payload, cb) {
  cb(null, '0x3f')
}
```
I hope it makes sense

Reply:
Updated